src/getting-started/infra/ai.bicep

```diff
@@ -19,7 +19,7 @@ var openaiDeployment = [
     model: {
       format: 'OpenAI'
       name: 'gpt-35-turbo'
-      version: '0613'
+      version: '0125'
```

Comment:
I'm not sure what the best answer here is. 0613 is old and doesn't support tool calls, but 0914, 1106, and 0125 are available in fewer regions.

Reply:
Hmm

https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/models#gpt-35-turbo-model-availability

Reply:
I think the best we can do is provide guidance some where in the readme/docs.

Reply:
https://github.com/dotnet/ai-samples/issues/28